xml/admin_operating_pools.xml

```diff
@@ -64,825 +65,821 @@
+        as the pool's usage because you can write to the same object many times.
+        The result is that the pool's usage will remain the same but the number
+        of bytes written to the pool will grow.
+       </para>
+      </listitem>
+     </varlistentry>
+     <varlistentry>
+      <term>USED COMPR</term>
+      <listitem>
+       <para>
+        Number of bytes allocated for compressed data.
+       </para>
+      </listitem>
+     </varlistentry>
+     <varlistentry>
+      <term>UNDER COMPR</term>
+      <listitem>
+       <para>
+        Number of bytes that the compressed data occupy when it is not
+        compressed.
+       </para>
+      </listitem>
+     </varlistentry>
+    </variablelist>
+   </sect2>
+
+   <sect2 xml:id="ceph-getting-pool-values">
+    <title>Getting pool values</title>
     <para>
-     This ensures that no object in the data pool will receive I/O with fewer
-     than <literal>min_size</literal> replicas.
+     To get a value from a pool, run the following <command>get</command>
+     command:
     </para>
-   </note>
-   <tip>
-    <title>Get the number of object replicas</title>
+ <screen>&prompt.cephuser;ceph osd pool get <replaceable>pool-name</replaceable> <replaceable>key</replaceable></screen>
     <para>
-     To get the number of object replicas, execute the following:
+     You can get values for keys listed in <xref linkend="ceph-pools-values"/>
+     plus the following keys:
     </para>
-<screen>&prompt.cephuser;ceph osd dump | grep 'replicated size'</screen>
+    <variablelist>
+     <varlistentry>
+      <term>pg_num</term>
+      <listitem>
+       <para>
+        The number of placement groups for the pool.
+       </para>
+      </listitem>
+     </varlistentry>
+     <varlistentry>
+      <term>pgp_num</term>
+      <listitem>
+       <para>
+        The effective number of placement groups to use when calculating data
+        placement. Valid range is equal to or less than
+        <literal>pg_num</literal>.
+       </para>
+      </listitem>
+     </varlistentry>
+    </variablelist>
+    <tip>
+     <title>All of a pool's values</title>
+     <para>
+      To list all values related to a specific pool, run:
+     </para>
+ <screen>
+ &prompt.cephuser;ceph osd pool get <replaceable>POOL_NAME</replaceable> all
+ </screen>
+    </tip>
+   </sect2>
+
+   <sect2 xml:id="ceph-pools-values">
+    <title>Setting pool values</title>
     <para>
-     &ceph; will list the pools, with the <literal>replicated size</literal>
-     attribute highlighted. By default, &ceph; creates two replicas of an
-     object (a total of three copies, or a size of 3).
+     To set a value to a pool, execute:
     </para>
-   </tip>
-  </sect2>
+ <screen>&prompt.cephuser;ceph osd pool set <replaceable>pool-name</replaceable> <replaceable>KEY</replaceable> <replaceable>VALUE</replaceable></screen>
+    <para>
+     The following is a list of pool values sorted by a pool type:
+    </para>
+    <variablelist>
```

Comment:
for this variable list, i cannot decide what is a cmdline option and what is a configuration option

Reply:
Unsure. We can look at this in the new year. This isn't new stuff. Doesn't look so bad in the web version: https://susedoc.github.io/doc-ses/master/single-html/ses-admin/#ceph-pools-values

Reply:
What would you suggest changes?